contrib/terraform/vsphere/cloud-init.yaml

```
@@ -0,0 +1,37 @@
+---
+# cloud-config
+
+# sets the hostname to 'terraformed'
+hostname: terraformed
+
+ssh_authorized_keys:
+- ${ssh_pub_key}
```

For the sake of uniformity, can we do this like for Exoscale: https://github.com/kubernetes-sigs/kubespray/blob/master/contrib/terraform/exoscale/modules/kubernetes-cluster/templates/cloud-init.tmpl#L16

```suggestion
%{ for ssh_public_key in ssh_public_keys ~}
  - ${ssh_public_key}
%{ endfor ~}
```